prepare/metrics/llm_as_judge/llamaguard.py

```diff
@@ -0,0 +1,37 @@
+from unitxt import add_to_catalog
+from unitxt.inference import (
+    IbmGenAiInferenceEngine,
+    IbmGenAiInferenceEngineParams,
+)
+from unitxt.llm_as_judge import LLMAsJudge
+
+model_list = [
+    "meta-llama/llama-3-8b-instruct",
```

Comment:
maybe we should conduct meta evaluation on a dataset you have and see if they are any good

Reply:
Yes, I plan to do that for xstest, where I have evaluation results from a few LLMs